### PR TITLE
Calling tracking

### PR DESCRIPTION
**Pull request by deanrobertcook**
Moves all of the call tracking out of SE and into the UI project, where it's a bit easier to listen to everything that's happening
#### APK
[Download build #8369](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8369/artifact/build/artifact/wire-dev-PR543-8369.apk)
[Download build #8372](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8372/artifact/build/artifact/wire-dev-PR543-8372.apk)
[Download build #8390](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8390/artifact/build/artifact/wire-dev-PR543-8390.apk)
[Download build #8392](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8392/artifact/build/artifact/wire-dev-PR543-8392.apk)
[Download build #8393](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8393/artifact/build/artifact/wire-dev-PR543-8393.apk)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -73,7 +73,7 @@ ext {
     playServicesVersion = '10.0.1'
 
     audioVersion = System.getenv("AUDIO_VERSION") ?: '1.205.0@aar'
-    zMessagingDevVersionBase = '90.1363'
+    zMessagingDevVersionBase = '90-SNAPSHOT'
```
Comment (deanrobertcook) [inline build.gradle]: Waiting on https://github.com/wireapp/wire-android-sync-engine/pull/55 before this PR can be merged

File: app/src/main/scala/com/waz/zclient/WireApplication.scala [hunk 2]
```diff
@@ -151,6 +160,10 @@ class WireApplication extends MultiDexApplication with WireContext with Injectab
     inject[MessageNotificationsController]
     inject[ImageNotificationsController]
     inject[CallingNotificationsController]
+
+    //TODO - is this early enough for app launch events?
```
Comment (hakonbo) [inline app/src/main/scala/com/waz/zclient/WireApplication.scala]: Perhaps add a ticket so we don't forget to check?

File: app/src/main/scala/com/waz/zclient/calling/CallingActivity.scala [hunk 3]
```diff
@@ -25,9 +25,10 @@ import android.view.ViewGroup.LayoutParams.MATCH_PARENT
 import android.view.{ViewGroup, WindowManager}
 import com.waz.threading.Threading
 import com.waz.zclient._
-import com.waz.zclient.calling.controllers.{GlobalCallingController, CurrentCallController}
+import com.waz.zclient.calling.controllers.{CurrentCallController, GlobalCallingController}
 import com.waz.zclient.calling.views.VideoCallingView
 import com.waz.zclient.common.controllers.PermissionActivity
+import com.waz.zclient.tracking.CallingTrackingController
```
Comment (hakonbo) [inline app/src/main/scala/com/waz/zclient/calling/CallingActivity.scala]: Unused import

File: app/src/main/scala/com/waz/zclient/messages/MessagesListAdapter.scala [hunk 4]
```diff
@@ -31,7 +31,7 @@ import com.waz.zclient.controllers.global.SelectionController
 import com.waz.zclient.messages.MessageView.MsgBindOptions
 import com.waz.zclient.messages.MessagesListView.UnreadIndex
 import com.waz.zclient.messages.RecyclerCursor.RecyclerNotifier
-import com.waz.zclient.tracking.TrackingController
+import com.waz.zclient.tracking.MainTrackingController$
```
Comment (hakonbo) [inline app/src/main/scala/com/waz/zclient/messages/MessagesListAdapter.scala]: Unused import

File: app/src/main/scala/com/waz/zclient/tracking/CallingTrackingController.scala [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.waz.zclient.tracking
+
+import android.content.Context
+import com.waz.ZLog.ImplicitTag._
+import com.waz.ZLog.verbose
+import com.waz.api.{CauseForCallStateEvent, NetworkMode, VoiceChannelState}
+import com.waz.model.ConvId
+import com.waz.service.call.CallInfo.ClosedReason.{AnsweredElsewhere, Interrupted, Normal}
```
Comment (hakonbo) [inline app/src/main/scala/com/waz/zclient/tracking/CallingTrackingController.scala]: Unused `AnsweredElsewhere` and `Interrupted`?

**Comment by xuanzhaopeng** [comment]
1) Attribute CALINDG_VERSION always be "C2", although I switch the calling from V2 to V3
2) Make a group call among 3 users, only C1 and C2 join the call, C3 do nothing. C1 and C2 all end the call. we do not see the "Max users during the call" in the event "calling.ended_call"

**Comment by xuanzhaopeng** [comment]
Checked track,
Checked video calling issue,
Checked crash on sign in.

All fixed.

**Pull request merged**